hashtheplanet/sql/db_connector.py

```diff
@@ -144,3 +142,12 @@ def get_all_hashs(session):
         stmt = select(Hash)
         hashs = session.execute(stmt).scalars().all()
         return hashs
+
+    @staticmethod
+    def get_static_files(session):
+        """
+        Returns all files ending with .txt or .html
+        """
+        return list(map(lambda element: element[0], session.query(File.path).filter(
+            File.path.regexp_match(r"([a-zA-Z0-9\s_\\.\-\(\):])+(.html|.md|.txt)$")).all()
+        ))
```

Comment:
And what about something like this ?

```suggestion
        static_files_query = session
            .query(File.path)
            .filter(File.path.regexp_match(r"([a-zA-Z0-9\s_\\.\-\(\):])+(.html|.md|.txt)$"))
        return [path for path, in static_files_query]
```